src/extension/types/typedefs.js

```diff
@@ -118,10 +118,10 @@
  * @prop {string} id The plugin ID (mandatory)
  * @prop {PluginButton} [button] A button configuration object (optional)
  * @prop {string} [container] The ID of a dropdown to add this plugin to (optional)
- * @prop {boolean} [override] Replace an existing plugin (optional)
```

Comment:
I decided to drop this unused legacy property. It has no more effect in the new sidekick.